.travis.yml

```diff
@@ -28,6 +28,8 @@ install:
 #  - pip install coverage
   - pip install flake8
   - pip install pylint
+  - pip install radical.pilot
+  - pip install radical.utils
```

Comment:
You do not need this, it gets installed with rp

Reply:
fixed in the last commit.